Gems/MotionMatching/Code/Source/FeatureMatrixMinMaxScaler.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <FeatureMatrix.h>
+#include <FeatureMatrixTransformer.h>
+
+namespace EMotionFX::MotionMatching
+{
+    class MinMaxScaler
+        : public FeatureMatrixTransformer
+    {
+    public:
+        AZ_RTTI(MinMaxScaler, "{95D5BBA7-6144-4219-82F0-34C2DAB7DD3E}", FeatureMatrixTransformer);
+        AZ_CLASS_ALLOCATOR_DECL
+
+        MinMaxScaler() = default;
+
+        bool Fit(const FeatureMatrix& in, const Settings& settings = {}) override;
+
+        // Normalize and scale the values.
```

Comment:
There may be an offset applied as well as the scaling i.e. x' = mx + b rather than just mx.